lua/autorun/server/sv_autoclean.lua

```diff
@@ -1,36 +1,57 @@
-	CreateConVar('cfc_autoclean','500',{FCVAR_ARCHIVE, FCVAR_NOTIFY },'Autocleans the server based on seconds given')
-	
+
+local ConVarFlags = {FCVAR_ARCHIVE, FCVAR_NOTIFY}
+local DefaultCleanIntervalInSeconds = "500"
+CreateConVar("cfc_autoclean", DefaultCleanIntervalInSeconds, ConVarFlags, "Autocleans the server based on seconds given")
+
+local BlacklistedEntityTypes = {
+    ["gmod_wire_turret"] = true,
+    ["gmod_turret"] = true
+}
+
+local CleanupCommands = {
+    ["r_cleardecals"] = true,
+    ["stopsound"] = true
+}
+
+local function notifyPlayers( count )
+    local message = "[CFC_Autoclean] Removed " .. tostring(count) .. "objects."
+
+    print(Message) -- and server
+    for _, ply in pairs( player.GetHumans() ) do
+        ply:ChatPrint(Message)
```

Comment:
@brandonsturgeon are we wanting to chat print this to plys?

Reply:
What are your thoughts?

Reply:
I thought it was printed to the chat anyways...?

Reply:
I don't see harm in it, personally.